service/src/main/java/bio/terra/workspace/app/configuration/external/AzureConfiguration.java

```diff
@@ -1,9 +1,13 @@
 package bio.terra.workspace.app.configuration.external;
 
+import com.azure.core.management.Region;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.stereotype.Component;
 
+/** Azure POC code */
+@Component
```

Comment:
not sure if this is needed? Little fuzzy on the annotation, but the documentation was saying I need this to use it in conjunction with `@Autowired` for injection into other classes